iceoryx_utils/test/moduletests/test_cxx_algorithm.cpp

```diff
@@ -66,3 +71,134 @@ TEST_F(algorithm_test, MinOfManyElements)
 {
     EXPECT_THAT(min(0.0123f, -91.12f, 123.92f, -1021.2f, 0.0f), Eq(-1021.2f));
 }
+
+TEST_F(algorithm_test, MergeTwoDisjunctNonEmptySortedContainers)
+{
+    vector<int64_t, 10U> a, b;
+
+    for (int64_t i = 0; i < 5; ++i)
+    {
+        a.emplace_back(i);
```

Comment:
could you add an offset to the container value and alternate it each test case. The stack is reused and a weird implementation could lie to you and just pretend to have it sorted and fake the `cxx::vector` .. I know, quite unrealistic but an offset would be easy to add